source/js/nav-newsletter.js

```diff
@@ -58,7 +58,9 @@ class NavNewsletter {
     elements.container.classList.add("expanded");
     elements.buttonDesktop.classList.add("active");
     document.addEventListener(`click`, this.closeFormClickHandler);
-    document.addEventListener("scroll", this.closeFormClickHandler);
+    document.addEventListener(`scroll`, this.closeFormClickHandler, {
```

Comment:
Let's file an issue to investigate whether we clean up these events in `closeFormClickHandler`, becuase we don't want a scroll handler sticking around =)

Reply:
Ticket filed: #4496